spec/models/post_spec.rb

```diff
@@ -1,7 +1,8 @@
 require "rails_helper"
 
 RSpec.describe Post, type: :model do
-  it { should belong_to(:submitter).dependent(:destroy) }
+  let(:post) { build(:text_post) }
+  it { expect(post).to have_many(:comments).dependent(:destroy) }
```

Comment:
nitpick: space between these 2 lines please

Reply:
line 5 could be `it { is_expected.to have_many(:comments).dependent(:destroy) }`